.github/workflows/functional-tests.yaml

```diff
@@ -0,0 +1,17 @@
+name: run acceptance tests
+
+on:
+  label:
+    types:
+      - post
```

Comment:
nice idea to run func test on tag, but is it clear to use post tag or maybe to run func tests after golangci-lint is passed?
and another issue how to run func tests properly on otc?